src/users/users.model.ts

```diff
@@ -0,0 +1,49 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { Column, DataType, Model, Table } from 'sequelize-typescript';
+
+interface UserCreationAttrs {
+  login: string;
+  password: string;
+  age: number;
+}
+
+@Table({ tableName: 'users' })
+export class User extends Model<User, UserCreationAttrs> {
+  @ApiProperty({ example: '1', description: 'Unique identificator' })
+  @Column({
+    type: DataType.INTEGER,
+    unique: true,
+    autoIncrement: true,
```

Comment:
Better use UUID, you can create UUID on BD due defaultValue: DataType.UUIDV4

Reply:
done